cornice_swagger/swagger.py

```diff
@@ -498,7 +501,8 @@ def get_tags_from_path(service):
 
     swag = CorniceSwagger(services, def_ref_depth=-1, param_ref=0)
     doc = swag(title, version, ignores=('head'),
-               default_tags=get_tags_from_path, **kwargs)
+               default_tags=get_tags_from_path,
+               summary_docstrings=True, **kwargs)
     doc.update(**kwargs)
```

Comment:
Only if you try to pass it to the `generate_swagger_spec` call, but that's not expected. All keyword arguments from `generate_swagger_spec` should be raw swagger attributes. I'm just keeping this call for backward compatibility, but we should drop it in the near feature.

The expected way to call the generator if you want to provide a custom value for summary docstrings is through the `CorniceSwagger` class call.